linkedin_v2/linkedin.py

```diff
@@ -187,10 +197,243 @@ def get_profile(self, member_id=None, member_url=None, selectors=None,
+                    {
+                        "entityLocation": "",
+                        "thumbnails": [
+                            {
+                                "resolvedUrl": ""
+                            }
+                        ]
+                    }
+                ],
+                "title": ""
+            }
+        }
+        if comment is not None:
+            post['comment'] = comment
+        if title is not None:
+            post['content']['title'] = title
+        if submitted_url is not None:
+            post['content']['submitted-url'] = submitted_url
+        if submitted_image_url is not None:
+            post['content']['contentEntities']['thumbnails']['resolvedUrl'] = submitted_image_url
+        response = application.make_request(
+            'POST', ENDPOINTS.SHARE, data=json.dumps(post))
+        raise_for_error(response)
+        return response.json()
+
+    def search_company(self, params):
+        url = '%s/search?q=companiesV2&%s' % (
+            ENDPOINTS.BASE, urllib.parse.urlencode(params))
+        response = self.make_request('GET', url)
+        # raise_for_error(response)
+        return response.json()
+
+    def get_organization(self, organization_id, params=None, headers=None):
+        url = '%s/organizations/%s' % (ENDPOINTS.BASE, organization_id)
+        response = self.make_request(
+            'GET', url, params=params, headers=headers)
+        raise_for_error(response)
+        return response.json()
+
+    def get_brand(self, brand_id, params=None, headers=None):
+        url = '%s/organizationBrands/%s' % (ENDPOINTS.BASE, brand_id)
+        response = self.make_request(
+            'GET', url, params=params, headers=headers)
+        raise_for_error(response)
+        return response.json()
+
+    def send_invitation(self, invitee_email):
+        post = {
+            "invitee": "urn:li:email:%s" % invitee_email,
+            "message": {
+                "com.linkedin.invitations.InvitationMessage": {
+                    "body": "Let's connect!"
+                }
+            }
+        }
+        response = application.make_request(
+            'POST', '%s/invitations' % ENDPOINTS.BASE, data=json.dumps(post))
+        raise_for_error(response)
+        return response.json()
+
+    def search_job(self):
+        url = '%s/recommendedJobs?q=byMember' % ENDPOINTS.BASE
+        response = self.make_request('GET', url)
+        # raise_for_error(response)
+        return response.json()
+
+    def get_job(self, **kwargs):
+        return self.search_job()
+
+    def get_post_comments(self, selectors, params=None, **kwargs):
+        url = '%s/socialActions/urn:li:share:%s/comments' % (
+            ENDPOINTS.BASE, kwargs['post_id'])
+        print(url)
+        response = self.make_request(
+            'GET', url, params=params)
+        # raise_for_error(response)
+        return response.json()
+
+    def get_groupss(self, group_id, params=None, headers=None):
```

Comment:
What's the difference between `get_groupss` and `get_groups`? At first, I thought it was a typo 😉but they seem to behave differently, in which case, would it be possible to rename this method so is more self-explanatory?